src/Mado/QueryBundle/Queries/QueryBuilderFactory.php

```diff
@@ -37,6 +37,11 @@ class QueryBuilderFactory extends AbstractQuery
 
     protected $select;
 
+    public function ensureFieldsDefinedPublic()
+    {
+        $this->ensureFieldsDefined();
```

Comment:
Is better to render ensureFieldsDefined public I think

Reply:
render?

Reply:
ah ok, sorry